Pipeline/pipeline.py

```diff
@@ -173,36 +191,34 @@ def feature_reduction(self, n):
 
         return self.X
 
-    def data_scaling(self, test_size, scaling=True, scalers=None):
+    def data_scaling(self, test_size, scaling=True, scalers=[]):
         """
 
+
         Parameters
         ----------
         test_size : float
             Percentage of data destined for testing.
         scalers : scikit-learn object
-            scikit-learn scalers
+            scikit-learn scalers.
         scaling : boolean, optional
             Choose between scaling the data or not.
             The default is True.
 
         Returns
         -------
-        x_train : array
-            Features destined for training.
-        x_test : array
-            Features destined for test.
-        y_train : array
-            Labels destined for training.
-        y_test : array
-             Labels destined for test.
+        Array
+            x_train - features destined for training.
+        Array
+             x_test -features destined for test.
+        Array
+            y_train - labels destined for training.
+        Array
+            y_test - labels destined for test.

```

Comment:
Format was following the numpydoc, now it is not. This should be reverted.

Reply:
Done!